knowledge-base/grid-rowcollection-nullreference.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Neither "rowcollection", nor "nullreference" exist as separate words like this in the error message. Using them in the URL is not optimal. Consider this:

`grid-nullreferenceexception-gridrowcollection.md`